jsonnet/kube-prometheus/components/prometheus.libsonnet

```diff
@@ -34,6 +34,9 @@ local defaults = {
     _config: {
       prometheusSelector: 'job="prometheus-' + defaults.name + '",namespace="' + defaults.namespace + '"',
       prometheusName: '{{$labels.namespace}}/{{$labels.pod}}',
+      thanosTargetGroup: {
+        namespace: defaults.namespace,
+      },
       thanosSelector: 'job="thanos-sidecar"',
```

Comment:
Seems like there might be more and more options that we would like to pass to thanos mixin. Maybe we should create a n additional field of `mixin._config.thanos` and pass all configuration options from there to the mixin? This way we won't need to redefine configuration fields. WDYT?

Reply:
I agree with you, adding `.thanos` property will be future proof and intuitive.